### ISSUE TITLE
Street lighting layer allows tagging ALL highways

### ISSUE THREAD

**Issue by HolgerJeromin** [streetcomplete/StreetComplete#4121]
Perhaps not really a bug. But at least this differs from streetlighting to sidewalks. 
`living_streets` are not shown/asked for sidewalks.

With lighting I am "supposed" to add lit to "every" highway. But currently `service=driveways` are excluded in the normal quests.

**Comment by westnordost** [comment]
> With lighting I am "supposed" to add lit to "every" highway. But currently service=driveways are excluded in the normal quests.

Correct, overlays are made for advanced users, the guidelines regarding for which elements it is "asked" are softened. Because it is not "asked", there is no quest pin. It is just marked as not having this tag.

However, for some elements, it is not shown if an information is missing. Most prominently, if a road is **private**. For the lit overlay also for indoor ways.

So, if a private/indoor way has a `lit=*` tag, it is shown on the map. If it has no `lit=*` tag, it is not highlighted at all. But you can still tap on it to specify it.
